language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/CommandClientImplTest.scala

```diff
@@ -128,76 +127,34 @@ class CommandClientImplTest
     List(command).asJava
   }
 
-  private type SubmitAndWait[A] =
-    (
-        String,
-        String,
-        String,
-        String,
-        Optional[Instant],
-        Optional[Duration],
-        Optional[Duration],
-        java.util.List[Command],
-    ) => Single[A]
-  private type SubmitAndWaitWithToken[A] =
-    (
-        String,
-        String,
-        String,
-        String,
-        Optional[Instant],
-        Optional[Duration],
-        Optional[Duration],
-        java.util.List[Command],
-        String,
-    ) => Single[A]
+  private type SubmitAndWait[A] = CommandsSubmission => Single[A]
```

Comment:
One more reason to like it. ❤️

Reply:
Btw since the type itself is so short, terse and understandable, I would probably not have an alias here.